common/frame.go

```diff
@@ -1973,6 +1965,31 @@ type frameExecutionContext interface {
 	ID() runtime.ExecutionContextID
 }
 
+func (f *Frame) runActionOnSelector(
+	ctx context.Context, selector string, strict bool, fn elementHandleActionFunc,
+) (bool, error) {
+	handle, err := f.Query(selector, strict)
+	if err != nil {
+		return false, fmt.Errorf("query: %w", err)
+	}
+	if handle == nil {
+		f.log.Debugf("Frame:runActionOnSelector:nilHandler", "fid:%s furl:%q selector:%s", f.ID(), f.URL(), selector)
+		return false, nil
+	}
+
+	v, err := fn(ctx, handle)
+	if err != nil {
+		return false, fmt.Errorf("function called: %w", err)
```

Comment:
`calling function` instead of `function called` to be consistent with the rest of our errors?

Reply:
Resolved in 9e1d3ed837aa954889a44780bd302c431e4b7b11